src/components/ProtectedRoute/ProtectedRoute.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Navigate, Outlet, useLocation } from 'react-router-dom';
+import { useAuth } from '../AuthContext';
```

Comment:
The path for this is incorrect  
```suggestion
import { useAuth } from '../auth/AuthContext';
```

Reply:
done. this is my commit id 01788ae27659a63ddd2949bc0c3f731a79607641